### PR TITLE
Waste Classification

### PR DESCRIPTION
**Pull request by Soumayan-pal01**
## Waste Classification

- The aim of this project is to recognize whether the image of the waste product is a Organic Waste or a Recyclable Waste

- [x] LGM-SOC'21 Participant
- [ ] DevIncept Participant 
- [ ] Contributor

Closes: #492

### Describe the add-ons or changes you've made

In this project we have used Convolutional neural Network approach to train a model on the provided images of Organic and Recyclable waste products. This model gives a Training accuracy of 98.61% and a Validation accuracy of 89.02% and also categorizes all the images of the evaluation set with correct labels.

## Type of change

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added things that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

**Review by prathimacode-hub — APPROVED** [review]
PR approved. As expected spot on and stunningly presented. 🙌🤩 @Soumayan-pal01
